src/js/ui/photoswipe-ui-default.js

```diff
@@ -59,14 +59,14 @@
 
 					// If allowLongCaptions is true, position caption just under picture and show "Expand" button if necessary
 					if (_options.allowLongCaptions) {
-						var imagePositionTop = item.initialPosition.y;
```

Comment:
It can't get `item.initialPosition.y` somehow

Reply:
`Uncaught TypeError: Cannot read property 'y' of undefined`

Reply:
This is a mystery since it was working for me earlier.

Reply:
I am only using the title attribute, could this cause an issue? (See the index.html)

Have a good flight!

Reply:
@andi34 I remembered to look at this again. In my rush to get out the door for our trip, I didn't understand the issue. My long caption solution has the captions - both the teaser and the full version - in <figcaption> elements (one with class="figure__caption--thumbnail" and the other "figure__caption--fullsize") using the structure shown here https://github.com/dimsemenov/PhotoSwipe/blob/master/website/documentation/getting-started.md#-how-to-build-an-array-of-slides-from-a-list-of-links  and then the Javascript harvests that information to create the options object. Aside from being easier (for me) to generate the HTML, instead of the JS that includes the options object, from the database , putting the information in the figcaption elements means that it can contain arbitrary HTML so links and even other images can be included. 

I'm going to update the documentation to make that clear and also check that I have not broken it for those using only the title object for the standard implementation. I wish I had done that earlier since now I can barely remember how it works!

I noticed in commit 83885edbbc9be003323254b5763ce129f4829eab that you moved some declarations further up. I had it where it was so that code would only be exercised if _options.allowLongCaptions was true.

Do you have a public-facing test page? When I ran your index.html I did not see any captions.

Reply:
Thanks for your reply!

There's a test page inside the source, only dist files haven't been generated.
You could clone my fork and checkout the "allow_long_captions-prettier" branch and generate them. Can do also and push to that branch if needed.

https://github.com/andi34/PhotoSwipe/blob/allow_long_captions-prettier/index.html

Without https://github.com/andi34/PhotoSwipe/commit/83885edbbc9be003323254b5763ce129f4829eab it wasn't working and had some errors visible in the browser console. My fork is far ahead of the official tree, have added a lot of stuff from the open Pullrequests.

Thanks a lot for looking at it, let me know if you need anything :)